frontend/app/services/TouchpointBackend.scala

```diff
@@ -60,10 +65,9 @@ object TouchpointBackend {
 
 case class TouchpointBackend(memberRepository: FrontendMemberRepository,
                              stripeService: StripeService,
-                             zuoraSoapService: ZuoraSoapService,
-                             zuoraRestService: ZuoraRestService) extends ActivityTracking {
-
-  val subscriptionService = new SubscriptionService(zuoraSoapService, zuoraRestService)
+			     zuoraSoapClient: soap.ClientWithFeatureSupplier,
+			     zuoraRestClient: rest.Client,
```

Comment:
I agree but as per discussions yesterday this should not block approval of the PR